README.md

```diff
@@ -1,12 +1,30 @@
 ![PureScript](https://github.com/purescript/purescript/raw/master/logo.png)
 
-Welcome to the PureScript documentation repository! This repository is meant to be a collaborative effort, so please feel free to add/edit content where it is appropriate to do so.
+Welcome to the PureScript documentation repository!
 
-## Introduction
-
-PureScript is a small strongly typed programming language that compiles to JavaScript.
+PureScript is a small, strongly typed programming language that compiles to JavaScript.
 To get a better overview of PureScript, visit [The PureScript Website](http://purescript.org).
 
+This repository is meant to be a collaborative effort, so please feel free to make a pull request to add/edit content or create an issue to discuss it. Without you, PureScript would have *no* documentation! Thanks for helping!
```

Comment:
`Without you, PureScript would have *no* documentation!` this phrasing sounds a little like a dig towards feature contributors not documenting their work, I doubt you meant it that way though.

Reply:
Yeah I have to say I'm not hugely fond of this sentence either. To me it reads as saying "we [contributors] haven't done the work of documenting PureScript, we're leaving it to everyone else", which... isn't really true - the evidence being the existence of this repo!

Reply:
But other than that it looks good to me too.

Reply:
@LiamGoodacre Well, I, myself, don't expect feature contributors to document their work. Adding features is hard enough! Expecting them to also come over here and add something seems a bit much to ask. I'm not much of a contributor to the PS compiler, myself, so I might be too lenient.

I'll update it - I see a better way to say what I meant.